docs/tutorials/quantum_walks.ipynb

```diff
@@ -62,22 +62,6 @@
     "</table>"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.        print(final)
</pre><p>can we move this to the last line, as otherwise everytime you run this function it prints the counter (see the output a few cells down)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5212/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>